src/app/jama/api/testruns.service.ts

```diff
@@ -797,6 +797,10 @@ export class TestrunsService {
 		if (startAt !== undefined) {
 			queryParameters = queryParameters.set('startAt', <any>startAt);
 		}
+
```

Comment:
Never ever change the generated files.
Instead change the call at ReporterDialog in order that, in the future, iterate through the different pages if is needed.

Reply:
Ok, I agree. I will iterate all the pages as it will be more than 50 Test Runs in a Test Cycles in some projects.